hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -20,6 +20,8 @@
     @Override
     public void configure() {
     	injectProperties();
+    	//The purpose is to set custom unique id for logging
+    	getContext().setUuidGenerator(new IDGenerator());
```

Comment:
We are setting transaction id in HNsecure. Do we have traceability of this message to HNClient?

Reply:
We don't but I'm not sure we want to build that dependency into our process. There is a possibility that vendors use HNSecure without a client.